src/Client/ConnectionPoolWithFailover.cpp

```diff
@@ -208,7 +208,10 @@ std::vector<ConnectionPoolWithFailover::TryResult> ConnectionPoolWithFailover::g
 
     bool fallback_to_stale_replicas = settings ? bool(settings->fallback_to_stale_replicas_for_distributed_queries) : true;
 
-    return Base::getMany(min_entries, max_entries, max_tries, try_get_entry, get_priority, fallback_to_stale_replicas);
+    return Base::getMany(min_entries, max_entries, max_tries,
```

Comment:
By the way, why is it not a property of the pool, like the other error-related parameters?

Reply:
I thought about this, but in this case you cannot change it for specific query (it is not that relevant for cap, since it this is about restoring from errors, so it should not affect the current query)